js/video/Example.svelte

```diff
@@ -46,6 +46,12 @@
 		border: 2px solid var(--border-color-primary);
 		border-radius: var(--radius-lg);
 		max-width: none;
+		overflow: hidden;
+	}
+	.container :global(video) {
+		width: var(--size-full);
+		height: var(--size-full);
+		object-fit: cover;
```

Comment:
I found the `object-fit` style of the video examples are different from the image ones:
https://github.com/gradio-app/gradio/blob/9a5811df9218b622af59ba243a937a9c36ba00f9/js/image/Example.svelte#L37

So fixed it for consistency.

As a result, the video example changes as below. (This example is `demo/kitchen_sink/run.py`)
<table>
<tr>
<th>This PR</th>
<th>main</th>
</tr>
<tr>
<td>
<img src="https://github.com/gradio-app/gradio/assets/3135397/1b16ad75-1084-41f7-bd60-edf34b7aaebd" />
</td>
<td>
<img src="https://github.com/gradio-app/gradio/assets/3135397/604e5a10-aeef-4270-8640-463b34ca48f9" />

</td>
</tr>

WDYT?